js/adapt-youtube.js

```diff
@@ -88,6 +91,29 @@ define(function(require) {
                 }
             }
         },
+        
+        onToggleInlineTranscript: function(event) {
+            if (event) event.preventDefault();
+            var $transcriptBodyContainer = this.$(".youtube-inline-transcript-body-container");
+            var $button = this.$(".youtube-inline-transcript-button");
+
+            if ($transcriptBodyContainer.hasClass("inline-transcript-open")) {
+                $transcriptBodyContainer.slideUp(function() {
```

Comment:
You've got [this bug](https://github.com/adaptlearning/adapt_framework/issues/1718) here, you can fix by doing:
```suggestion
                $transcriptBodyContainer.stop(true, true).slideUp(function() {
```